server/infra/database/knex.js

```diff
@@ -46,9 +46,11 @@ const knexConfig = {
 const knexConfigLegacyDB = {
```

Comment:
Current code requires a legacy DB url to be provided. I tried to check for it's existence first to avoid errors but the config is required.  Is that a requirement we want?

Reply:
The legacy DB is pretty much the same URL except it connects to the public schema, while the other connection if connected to the field_data schema. It is needed in the service because the field_data service communicates with the public schema (legacy db) when creating new raw captures.